### ISSUE TITLE
Clicking OK removes spaces

### ISSUE THREAD

**Issue by Saltibarsciai** [benwinding/quill-html-edit-button#45]
![29 04 2021_18 46 53_REC](https://user-images.githubusercontent.com/57238127/116579975-9beace80-a91b-11eb-8971-65beaaf82dcc.gif)

Version:
 "quill-html-edit-button": "^2.2.4",
 "vue2-editor": "^2.6.6",

usage:

//scripts
```
import htmlEditButton from "quill-html-edit-button";
Quill.register("modules/htmlEditButton", htmlEditButton);
...

 editor_settings: {
                    modules: {
                        htmlEditButton,
                    }
 }
```

//vue template
```
<vue-editor v-model="form.human_body" :editor-toolbar="custom_toolbar"  :editor-options="editor_settings" ></vue-editor>
```

**Comment by Saltibarsciai** [comment]
didn't know how to reopen #18, so just created new issue

**Comment by benwinding** [comment]
Thanks for reporting the issue @Saltibarsciai 

The library has been fixed, and some tests added to ensure it parses correctly back into the Quill editor. Should be available in version `2.2.5`

Let me know if there's any other issues.

Cheers,
Ben

**Comment by Saltibarsciai** [comment]
Thanks @benwinding works after update! If I'll discover new issues while using it I'll let you know
